flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesHaServices.java

```diff
@@ -42,7 +42,7 @@
 import static org.apache.flink.util.Preconditions.checkNotNull;
 
 /**
- * An implementation of the {@link AbstractHaServices} using Apache ZooKeeper.
+ * An implementation of the {@link AbstractHaServices} based on kubernetes API.
```

Comment:
```suggestion
 * An implementation of the {@link AbstractHaServices} using Kubernetes.
```

Reply:
Thanks for review. I have modified according to your suggestion.